deploy_flask.sh

```diff
@@ -8,7 +8,9 @@ apt install -y python3-pip
 pip3 install --upgrade pip
 pip3 install Flask 
 
-#go to the directory containing the file we want to run
-cd /vagrant/
 
-FLASK_APP=hello.py flask run --host=0.0.0.0
+
+FLASK_APP=/vagrant/hello.py
+sudo python3 hello.py >> /vagrant/log.txt 2>&1 &
```

Comment:
Why do you need `sudo` here? Previously the application ran with normal user privileges, I don't see an explanation for the need to elevate to root provoledges in the commit message. 

Also note that the `pip` commands above install dependencies into the user environment (Since they are running without `sudo`) so you are essentially running the app in a different environment them the one you've installed your dependencies into.

Since you're running inside a Vagrant VM, you should probably emulate what a production deployment would look like by using [wod_wsgi](https://modwsgi.readthedocs.io/en/develop/) with Apache for example.

Reply:
Vagrant executes the provisioning as a root user. sudo is not required.

Reply:
@lioramilbaum I changed it 10 days ago maybe  did something wrong but in the commit page you can see "Delete sudo command" 
So what am I missing here?

Reply:
@nofaryung I didn't look at the code. This conversion should have been acked or +1 and resolved by now.